src/System.Globalization.Calendars/tests/Misc/Calendars.netstandard1.7.cs

```diff
@@ -0,0 +1,93 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+using System.Globalization;
+using System.Collections.Generic;
+
+namespace System.Globalization.Tests
+{
+
+    public static class CalendarsMoreTests
```

Comment:
Calendars_netstandard17 if that's the defining characteristic of why these tests are here?